### PR TITLE
Fixes donkpocket and honkpocket cooling duration

### PR DESCRIPTION
**Pull request by TTerc**
[bug]

## About the PR 

Sets the donkpocket and honkpocket cooling duration to 420 seconds and makes the spawned honkpockets actually start the cooling timer

## Why's this needed? 

Lets people experience the horrors of cold honk pockets. And fixes a bug

### CODE REVIEW

File: code/modules/food_and_drink/snacks.dm [hunk 1]
```diff
@@ -813,6 +813,11 @@
 		name = "warm honk-pocket"
 		warm = 1
 
+		New()
+			..()
+			src.cooltime()
+			return
```
Comment (ZeWaka) [inline code/modules/food_and_drink/snacks.dm:819]:
```suggestion
```

File: code/modules/food_and_drink/snacks.dm [hunk 2]
```diff
@@ -813,6 +813,11 @@
 		name = "warm honk-pocket"
 		warm = 1
 
+		New()
+			..()
```
Comment (ZeWaka) [inline code/modules/food_and_drink/snacks.dm:817]:
```suggestion
			. = ..()
```

**Review by ZeWaka — APPROVED** [review]
note, they were already at 420 seconds....

**Comment by TTerc** [comment]
> note, they were already at 420 seconds....

I was under impression that it's not the case because when I was testing it, setting it to 420 made it last less than 42 seconds
Either way, I think it clarifies the duration time better

**Comment by ZeWaka** [comment]
> > note, they were already at 420 seconds....
> 
> I was under impression that it's not the case because when I was testing it, setting it to 420 

`420 SECONDS` == `4200`

**Pull request merged**